### PR TITLE
doc : Config JP text translate

### PR DESCRIPTION
**Pull request by wallsman**
いくらか抜けていた部分や翻訳抜けを修正。
Some omissions and missing translations were corrected.

### CODE REVIEW

File: src/MaaWpfGui/Res/Localizations/ja-jp.xaml [hunk 1]
```diff
@@ -37,15 +38,21 @@
...
-    <system:String x:Key="OriginiumShardAutoReplenishment">源石の欠片は自動で補充する</system:String>
-    <system:String x:Key="CustomInfrastEnabled">カスタムされた基地構成を有効にする（Alpha）</system:String>
+    <system:String x:Key="DormFilterNotStationedEnabled">作業中の人員を宿舎に移動しない（例:訓練所でのアイリーニなど）</system:String>
+    <system:String x:Key="OriginiumShardAutoReplenishment">源石の欠片を自動で補充</system:String>
+    <system:String x:Key="CustomInfrastEnabled">カスタムされた基地構成を有効にする（ベータ）</system:String>
+    <system:String x:Key="DefaultInfrast">標準設定</system:String>
+    <system:String x:Key="UserDefined">ユーザー定義</system:String>
+    <system:String x:Key="153_3">153 3周/日</system:String>
+    <system:String x:Key="243_3">243 3周/日<</system:String>
+    <system:String x:Key="243_4">243 4周/日<y</system:String>
```
Comment (ABA2396) [inline src/MaaWpfGui/Res/Localizations/ja-jp.xaml:48]: There is a typo here, there is an extra <y

**Comment by ABA2396** [comment]
Here is a [tool](https://github.com/MaaAssistantArknights/I18nResourceEditor/releases )  that can display translations in multiple languages at the same time, although its save function is currently a bit problematic, so it's best to avoid using it if possible, but it can be used to identify those statements that have not yet been translated.

File: src/MaaWpfGui/Res/Localizations/ja-jp.xaml (hunk 1 above)
Comment (wallsman) [reply src/MaaWpfGui/Res/Localizations/ja-jp.xaml]: https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/3774/commits/5f8a6ef0b2747e7e96b56e24378d70512d69e250

Thx. fix it.

File: src/MaaWpfGui/Res/Localizations/ja-jp.xaml [hunk 2]
```diff
@@ -37,15 +38,21 @@
...
+    <system:String x:Key="DormFilterNotStationedEnabled">作業中の人員を宿舎に移動しない（例:訓練所でのアイリーニなど）</system:String>
+    <system:String x:Key="OriginiumShardAutoReplenishment">源石の欠片を自動で補充</system:String>
+    <system:String x:Key="CustomInfrastEnabled">カスタムされた基地構成を有効にする（ベータ）</system:String>
+    <system:String x:Key="DefaultInfrast">標準設定</system:String>
+    <system:String x:Key="UserDefined">ユーザー定義</system:String>
+    <system:String x:Key="153_3">153 3周/日</system:String>
+    <system:String x:Key="243_3">243 3周/日<</system:String>
+    <system:String x:Key="243_4">243 4周/日<</system:String>
+    <system:String x:Key="252_3">252 3周/日<</system:String>
+    <system:String x:Key="333_3">333 3周/日<</system:String>
```
Comment (ABA2396) [inline src/MaaWpfGui/Res/Localizations/ja-jp.xaml:50]:
```suggestion
    <system:String x:Key="243_3">243 3周/日</system:String>
    <system:String x:Key="243_4">243 4周/日</system:String>
    <system:String x:Key="252_3">252 3周/日</system:String>
    <system:String x:Key="333_3">333 3周/日</system:String>
```

Comment (wallsman) [reply src/MaaWpfGui/Res/Localizations/ja-jp.xaml]: https://github.com/MaaAssistantArknights/MaaAssistantArknights/pull/3774/commits/49a6192e4d6a81fe7eb8647a21cf89817756e557

Embarrassing...

**Pull request merged**